lib/tasks/foreman_openscap_tasks.rake

```diff
@@ -44,6 +44,12 @@ namespace :foreman_openscap do
     end
     Rake::Task['rubocop_foreman_openscap'].invoke
   end
+
+  desc "Clean duplicate messages for ArfReport"
+  task :clean_messages => :environment do
+    require 'foreman_openscap/message_cleaner'
```

Comment:
require should be in global scope, dynamic dependencies reveal problems too late in runtime. Unless there's some problem with requiring the `MessageCleaner` object definition elsewhere, I'd recommend moving it outside from here.